includes/class-sensei-frontend.php

```diff
@@ -1067,72 +1067,6 @@ public function sensei_course_category_main_content() {
 
 	} // End sensei_course_category_main_content()
 
-	public function sensei_login_form() {
-		?>
-		<div id="my-courses">
-			<?php Sensei()->notices->maybe_print_notices(); ?>
-			<div class="col2-set" id="customer_login">
-
-				<div class="col-1">
-					<?php
-					// output the actual form markup
-                    Sensei_Templates::get_template( 'user/login-form.php');
-					?>
-				</div>
-
-			<?php
-			if ( get_option('users_can_register') ) {
-
-				// get current url
-				$action_url = get_permalink();
```

Comment:
Where is the server-side code that’s handling this form? Should we probably remove it, too?